src/track/taglib/trackmetadata_common.cpp

```diff
@@ -188,6 +211,16 @@ bool parseSeratoMarkers2(
     return parseSeratoMarkers2(pTrackMetadata, toQByteArrayRaw(byteVec), fileType);
 }
 
+TagLib::String dumpSeratoBeatGrid(
+        const TrackMetadata& trackMetadata,
+        FileType fileType) {
+    const QByteArray utf8Data =
+            trackMetadata.getTrackInfo().getSeratoTags().dumpBeatGrid(fileType);
+    return TagLib::String(
```

Comment:
This can make use of a comment about the format of data we handle here.

Reply:
Do we have here the base64 format? I think it would also helpful to append the dump() function with a to... prefix to make it more maintainable.

Reply:
Can you elaborate what you mean? I'm not sure. The Base64 encoding is only used if `fileType` is FLAC or MP4, not MP3.

Reply:
utf8Data can be everything. I don't even think it is actually utf8. So this is equivalent to "var1" or such. 
A bit info about the content and content format is helpful here.